wpilibj/src/main/java/edu/wpi/first/wpilibj/PneumaticsBase.java

```diff
@@ -80,24 +80,110 @@ static int getDefaultForType(PneumaticsModuleType type) {
    */
   void setOneShotDuration(int index, int durMs);
 
+  /**
+   * Returns whether the compressor is active or not.
+   *
+   * @return True if the compressor is on - otherwise false.
+   */
   boolean getCompressor();
 
+  /**
+   * Returns the state of the pressure switch.
+   *
+   * @return True if pressure switch indicates that the system is not full, otherwise false.
```

Comment:
Sure this isnt the other way around?

Reply:
yep, original compressor docs said "true if the pressure is low"